todoman/cli.py

```diff
@@ -48,6 +48,26 @@ def _validate_date_param(ctx, param, val):
         raise click.BadParameter(e)
 
 
+def _validate_start_date_param(ctx, param, val):
+    if val is None:
+        return val
+    if val.startswith('before '):
+        is_before = True
+        val = val[len('before '):]
+    elif val.startswith('after '):
+        is_before = False
+        val = val[len('after '):]
+    else:
+        raise click.BadParameter(
+            'The start date of the task should be'
+            'in format \'before <date-format>\'')
```

Comment:
`'in format "[before|after] <date>"'`. Note that you're not mentioning `after`, and it's simpler ("prettier") to user `"` inside strings.